tasks-by-users/mmusk/Creative Card/style.css

```diff
@@ -0,0 +1,266 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;700&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+
+  font-family: 'Poppins', sans-serif;
+}
+
+body{
+    display: flex;
+    justify-content: center;
+    align-content: center;
+
+    min-height: 100vh;
+
+    background-color: #131313;
+}
+
+.container{
+    position: relative;
+}
+
+.container .card{  
+    position: relative;
+
+    width: 320px;
+    height: 450px;
+
+    background-color: #232323;
+
+    border-radius: 20px;
+
+    overflow: hidden;
+}
+
+.container .card::before{
+    content: "";
+
+    position: absolute;
+
+    top: 0;
+    left: 0;
+
+    width: 100%;
+    height: 100%;
+
+    background-color: #9bdc28;
+
+    clip-path: circle(150px at 80% 20%);
+
+    transition: 0.5s ease-in-out;
+}
+
+.container .card:hover::before{
+
+    clip-path: circle(300px at 80% -20%);
+
+}
+
+.container .card::after{
+    content: "Nike";
+
+    position: absolute;
+
+    top: 30%;
+    left: -20%;
+
+    width: 100%;
+    height: 100%;
+
+    color: rgba(224, 224, 224, 0.04);
+
+    font-size: 12em;
+    font-weight: 800;
+    font-style: italic;
+}
+
+.container .card .imgbx{
+
+    position: absolute;
+
+    width: 100%;
+    height: 220px;
+
+    top: 50%;
+
+    z-index: 10;
+
+    transform: translateY(-50%);
+    transition: 0.5s;
+}
+
+.container .card .imgbx:hover{
+    top: 0%;
+
+    transform: translateY(0%);
+
+    opacity: 0.9;
+}
+
+.container .card .imgbx img{
+    position: absolute;
+
+    top: 50%;
+    left: 50%;
+
+    transform: translate(-50%, -50%) rotate(-25deg);
+
+    width: 270px;
+}
+
+.container .card .contentbx{
+    position: absolute;
+
+    bottom: 0;
+
+    width: 100%; 
+    height: 100px;
+
+    text-align: center;
+
+    z-index: 10;
+
+    transition: 1s;
+}
+
+.container .card:hover .contentbx{
+    height: 210px;
+}
+
+.container .card .contentbx h2{
+    position: relative;
+
+    color: #fff;
+
+    font-weight: 600;
+
+    letter-spacing: 1px;
+
+}
+
+.container .card .contentbx .size,
+.container .card .contentbx .colour{
+    padding: 8px 20px;
+
+    align-items: center;
+    justify-content: center;
+    display: flex;
+
+    transition: 0.5s;
+
+    opacity: 0;
+
+    visibility: hidden;
+}
+
+.container .card:hover .contentbx .size{
+    opacity: 1;
```

Comment:
i think this is not needed it will have default  `opacity: 1 `

Reply:
It is needed or else the size option gets disappered @coder-mezab